packages/types/lib/environment/db.ts

```diff
@@ -46,6 +46,9 @@ export interface DBEnvironment extends Timestamps {
 
     webhook_receive_url: string | null;
     otlp_settings: { endpoint: string; headers: Record<string, string> } | null;
+
+    deleted: boolean;
+    deleted_at?: Date | null;
```

Comment:
can't be undefined
```suggestion
    deleted_at: Date | null;
```

Alternatively you can change the extends to use `TimestampsAndDeletedCorrect`

Reply:
Good to know